index.html

```diff
@@ -2,15 +2,117 @@
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+    <div class="nav-bar">
+        <ul class="nav-menu">
+            <li class="nav-item">
+                <a href="#" class="nav-link">Home</a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link">Christmas</a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link">Cakes</a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link">Cupcakes</a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link">Wedding</a>
+            </li>
+        </ul>
+    </div>
+    <section class="welcome-intro">
+        <h2>Hello there!</h2>
+        <p class="welcome text1">
+            Here at Creamy Creations you can choose from our many ready-made designs or get creative and create your own bespoke showpiece! Everything is fresh, made from quality ingredients to give you the taste you deserve for any occasion.
+        </p>
+        <p class="welcome text2">
+            With the holidays approaching explore our christmas creations and much more. Then you can pick the option to deliver to your door or collect from our store!
+        </p>
+    </section>
+
+    <hero class="product-container hero">
+        <img class="feature-img" src="./images/xmas.jpg" alt="man working behind the counter of a cake shop" />
+        <div class="overlay">
+            <h1 class="product-title">Christmas</h1>
+        </div>
+    </hero>
+
+    <section class="product-container birthday">
+        <img class="bday-img" src="./images/three-cakes.jpg" alt="" />
+        <div class="overlay">
+            <h1 class="product-title">Birthday</h1>
+        </div>
+    </section>
+
+    <section class="product-container cupcakes">
+        <img class="cupcake-img" src="./images/flower-cupcakes.jpg" alt="" />
+        <div class="overlay">
+            <h1 class="product-title">Cupcakes</h1>
+        </div>
+    </section>
+
+    <section class="product-container wedding">
+        <img class="wedding-cake" src="./images/rose-cake.jpg" alt="" />
+        <div class="overlay">
+            <h1 class="product-title">Wedding</h1>
+        </div>
+    </section>
+
+    <section class="product-container gluten-vegan">
+        <img class="vegan" src="./images/fruit-tart.jpg" alt="" />
+        <div class="overlay">
+            <h1 class="product-title">Gluten Free / Vegan</h1>
+        </div>
+    </section>
+
+    <footer class="footer-container">
+        <div class="footer-links">
+            <a>Contact us</a>
+            <a href="#">Stores</a>
+            <a href="#">Delivery</a>
+        </div>
+
+        <div class="newsletter-container">
+            <h5>Sign up to our newsletter</h5>
+
+            <form class="newsletter">
```

Comment:
Great addition of the form. It's really nice to see you including everything you've learned so far